develop-docs/sdk/javascript/browser-tracing.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Tracing in the Browser
+description: Interesting aspects of tracing and performance instrumentation in the browser SDKs.
+sidebar_order: 10
+---
+
+The tracing behavior in our browser SDKs is somewhat unique and significantly differs from tracing in the backend. This page collects the most important aspects.
+
+<Note>
+  The aspects described in this document apply to all browser SDKs. This
+  includes `@sentry/browser` and all SDKs building on top of it like
+  `@sentry/react`, `@sentry/angular` or `@sentry/vue` but also the client-side
+  parts of meta frameworks like `@sentry/nextjs`.
+</Note>
+
+Please note that any kind of automatic tracing instrumentation in the browser requires the `browserTracingIntegration()` to be added to the SDK configuration.
+The default configuration of the SDK does not include any performance or tracing-related instrumentation to save on [bundle size](../bundle-size) for errors-only users.
+
+## Pageload and Navigation Spans
+
+Browser SDKs automatically create spans for the initial pageload as well as for subsequent soft navigations.
+
+These spans however don't behave like conventional spans. We call them **idle spans** because they are started at some point (more on that later) but not explicitly ended.
```

Comment:
```suggestion
These spans however don't behave like conventional spans. We call them **idle spans** because they are started at some point (more on that later) but are not explicitly ended.
```